modules/ROOT/nav.adoc

```diff
@@ -157,7 +157,8 @@ include::partial$nav-dw.adoc[]
 ** xref:configuring-mule-stacktraces.adoc[Configuring Mule Stack Traces]
 * xref:reliability-patterns.adoc[Reliability Patterns]
 * xref:mule-4-api-javadoc.adoc[Mule Runtime Engine Javadoc]
-* xref:modularizing-your-configuration-files-for-team-development[Modularizing Your Configuration Files for Team Developent in Mule]
+* xref:modularizing-your-configuration-files-for-team-development[Modularizing Your Configuration Files for Team Development in Mule]
```

Comment:
```suggestion
* xref:modularizing-your-configuration-files-for-team-development.adoc[Modularizing Your Configuration Files for Team Development in Mule]
```
Without the `.adoc` extension, this will be a broken reference.